internal/ethapi/api.go

```diff
@@ -81,19 +81,19 @@ func (s *PublicEthereumAPI) MaxPriorityFeePerGas(ctx context.Context) (*hexutil.
 }
 
 type feeHistoryResult struct {
-	OldestBlock  rpc.BlockNumber  `json:"oldestBlock"`
+	OldestBlock  *hexutil.Big     `json:"oldestBlock"`
```

Comment:
Btw, the spec seems wrong. Can you also open a PR to fix the spec? https://github.com/ethereum/eth1.0-specs/blob/bdabd69d0943e518d9bdadf1ef253ae3f72368ce/json-rpc/spec.json#L248

Reply:
Updated: https://github.com/ethereum/eth1.0-specs/pull/279

Reply:
Thanks for fixing the spec too, it is indeed unnecessary to use `rpc.BlockNumber` for the return value.